### ISSUE TITLE
Ability To Specify continueUrl for password reset

### ISSUE THREAD

**Issue by mdrideout** [firebase/firebaseui-web#526]
**Is your feature request related to a problem? Please describe.**
When one sends a verification email, for email / password verification, one can specify a "continueUrl" - on the email verification confirmation page, this renders a button linking the user back to a specific page on the website.

I would like to request this option be available for the password reset functionality built into firebaseui.

**Describe the solution you'd like**
The config for the email verification is as follows:
```
var actionCodeSettings = {
    url: "https://www.example.com",
};

await firebaseAuth().currentUser.sendEmailVerification(actionCodeSettings);
```

For password resets, this URL could be specified in the firebaseui config where other settings are specified for email / password auth.
```
ui.start('#firebaseui-auth-container', {
  signInOptions: [
    {
      provider: firebase.auth.EmailAuthProvider.PROVIDER_ID,
      requireDisplayName: false,
      continueUrl: "https://www.example.com/"
    }
  ]
});
```

**Describe alternatives you've considered**
Alternatives are to not use Firebaseui, because firebase ui essentially takes over where forgotten password resets can be handled with its interface. 

**Additional context**
Google references documentation for this capability [here](https://firebase.google.com/docs/auth/web/manage-users#send_a_password_reset_email) and [here](https://firebase.google.com/docs/auth/web/passing-state-in-email-actions)

**Comment by bojeil-google** [comment]
Thanks for the request. We definitely want to provide the ability to set the `ActionCodeSettings` for password reset flows, not only for setting the continue URL but also for specifying whether you also want to handle the link from a mobile app.

**Comment by deepakputhraya** [comment]
@bojeil-google Are there any plans to implement this feature? Any suggested workarounds to add `continueUrl`? Any way I can help with this feature implemented?

**Comment by bojeil-google** [comment]
Hey @deepakputhraya, thanks for your interest. We are currently working on extending support for Apple sign-in. That is time sensitive to many customers. Let's revisit this after.

**Comment by jits** [comment]
Are there plans to support this any time soon? Currently the reset password flows are not ideal. Anything that could be done to speed this up? I'm happy to contribute a PR, with some guidance.

**Comment by rosalyntan** [comment]
This is being tracked internally at b/168630895. However, if you would like to contribute a PR, our contribution guide can be found here: https://github.com/firebase/firebaseui-web/blob/master/CONTRIBUTING.md.

**Comment by RafaelMiquelino** [comment]
> This is being tracked internally at b/168630895. However, if you would like to contribute a PR, our contribution guide can be found here: https://github.com/firebase/firebaseui-web/blob/master/CONTRIBUTING.md.

Is there any plan for this to be released any time soon?

**Comment by malcolmdeck** [comment]
I don't think that we have a timeline for implementation of this particular feature request at this time. If that changes, we'll try to let you know.

**Comment by dackers86** [comment]
Thank you for taking the time to submit this request.

This is a great idea and definitely something we can consider for the future. It is not currently prioritised for our immediate new version roadmap, as we are focusing on a new upcoming release.

In the meantime, we will keep this issue open so we can revisit it during a later planning cycle, particularly as we gather more feedback from the community.
